lib/docker/template/cache.rb

```diff
@@ -45,9 +43,10 @@ def aliased_context(builder)
       # ----------------------------------------------------------------------
 
       def cleanup(repo)
+        return unless repo.clean_cache?
```

Comment:
Any reason for this surface change? It provides no benefit in uncomplex methods to short-circuit.